service/src/test/java/bio/terra/workspace/common/utils/MockMvcUtils.java

```diff
@@ -22,6 +22,8 @@ public class MockMvcUtils {
   public static final String ADD_USER_TO_WORKSPACE_PATH_FORMAT =
       "/api/workspaces/v1/%s/roles/%s/members";
   public static final String CLONE_WORKSPACE_PATH_FORMAT = "/api/workspaces/v1/%s/clone";
+  public static final String UPDATE_WORKSPACES_V1_PROPERTIES_PATH_FORMAT =
```

Comment:
Could you rename to just WORKSPACE_V1_PROPERTIES_PATH_FORMAT so it can be shared with deleteproperties

Reply:
agree! When I write it, I also really hesitate on it due to all other names with a verb at beginning. And I supposed the UPDATE can represent both modify update and delete update. but your way is more generic. Thanks for the confirming.